app/js/services/sailsResponseProcessor.js

```diff
@@ -0,0 +1,37 @@
+const _ = require('lodash');
+
+function sailsResponseProcessor() {
+  'ngInject';
```

Comment:
you're not injecting anything into this service, so you probably don't need the `'ngInject'`